### PR TITLE
Reenable Omnia integration test

### PR DESCRIPTION
**Pull request by heyealex**
### Submission Checklist

* [x] Have you installed and run this change against pre-commit? (`pre-commit
  install`)
* [x] Are all tests passing? (`make tests`)
* [x] Have you written unit tests to cover this change?
* [x] Is unit test coverage still above 80%?
* [x] Have you updated all applicable documentation?
* [x] Have you followed the guidelines in our Contributing document?

### CODE REVIEW

**Comment by heyealex** [comment]
The Test Group 3 Integration tests containing the Omnia test passed, though it's not updating in the github check. 
[test status](https://console.cloud.google.com/cloud-build/builds;region=global/79f63f4c-9baa-4228-a62c-7401cd65bd2c;step=3?project=hpc-toolkit-dev)

File: tools/cloud-build/daily-tests/integration-group-3.yaml [hunk 1]
```diff
@@ -66,25 +66,25 @@ steps:
...
 
-#     ansible-playbook tools/cloud-build/daily-tests/ansible_playbooks/base-integration-test.yml \
-#       --user=sa_106486320838376751393 --extra-vars="project=${PROJECT_ID} build=$${BUILD_ID_SHORT}" \
-#       --extra-vars="@tools/cloud-build/daily-tests/tests/omnia.yml"
+    ansible-playbook tools/cloud-build/daily-tests/ansible_playbooks/base-integration-test.yml \
+      --user=sa_106486320838376751393 --extra-vars="project=${PROJECT_ID} build=$${BUILD_ID_SHORT}" \
+      --extra-vars="@tools/cloud-build/daily-tests/tests/omnia.yml"
 
 ## Test DDN Lustre with new VPC
 - id: lustre-new-vpc
```
Comment (cboneti) [inline tools/cloud-build/daily-tests/integration-group-3.yaml:90]: Here you have to make lustre-new-vpc wait for omnia.

File: tools/cloud-build/daily-tests/tests/omnia.yml [hunk 2]
```diff
@@ -19,6 +19,6 @@ zone: us-central1-c
 workspace: /workspace
 blueprint_yaml: "{{ workspace }}/community/examples/omnia-cluster.yaml"
 blueprint_dir: blueprint-omnia
-network: "default"
```
Comment (cboneti) [inline tools/cloud-build/daily-tests/tests/omnia.yml:22]: It used to use the default network.  I am wondering if the change was intentional...

**Review by cboneti — CHANGES_REQUESTED** [review]

File: tools/cloud-build/daily-tests/integration-group-3.yaml (hunk 1 above)
Comment (tpdownes) [reply tools/cloud-build/daily-tests/integration-group-3.yaml:90]: starting a new branch and running `git revert d35fe41a2fa3a13e9ce1501eb89d5de7303747af` would be one way of doing this in an exact way. If you also wanted to introduce the change below, you could do it as a separate commit.

Comment (cboneti) [reply tools/cloud-build/daily-tests/integration-group-3.yaml:90]: not sure if there was any misunderstanding here, but I just meant adding a line under 
waitFor: 
 - omnia

**Pull request closed without merging**